ui/src/components/App/App.test.tsx

```diff
@@ -3,8 +3,8 @@ import App from '.';
 import { shallow } from 'enzyme';
 import renderer from 'react-test-renderer';
 import { FakeHub } from '../../api/testutil';
-import CategoryFilter from '../CategoryFilter/CategoryFilter';
 import { createProviderAndStore } from '../../store/root';
+import LeftPane from '../LeftPane/LeftPane';
```

Comment:
how about renaming `LeftPane.tsx` to index.tsx so that import reads 
`import LeftPane from '../LeftPane';`